Gemfile.lock

```diff
@@ -186,7 +186,7 @@ GEM
       xcpretty-travis-formatter (>= 0.0.3)
     fastlane-plugin-appcenter (1.11.1)
     fastlane-plugin-sentry (1.11.0)
-    fastlane-plugin-wpmreleasetoolkit (9.0.0)
+    fastlane-plugin-wpmreleasetoolkit (9.0.1)
```

Comment:
This was to address the `po` generation regression that removed `\n` when not appropriate.